cdm/core/src/test/java/ucar/nc2/ft2/simpgeometry/TestSimpleGeometryFeature.java

```diff
@@ -1,93 +0,0 @@
-package ucar.nc2.ft2.simpgeometry;
```

Comment:
Was it not possible to save this test given the changes?

Reply:
Not sure how/why that got removed. Now erstored, and seems to pass.